### DISCUSSION TITLE
Automatically turn off "Highlight path issues" mode after closing all staff windows

### DISCUSSION THREAD

**Discussion by J-PIE-314** [OpenRCT2/OpenRCT2#18456] [Feature requests]
Current:
After turning on "Highlight path issues" mode. I will move and organize handymen as needed. Once done, I promptly use the keyboard shortcut "shift-backspace" to clear the screen. The view remains on "Highlight path issues" mode.

Feature Request:
I would like to request that if a handyman window or main staff window is opened during "Highlight path issues" mode then the mode be automatically turned off once all the aforementioned windows are closed. (Similar to how "Underground/Inside View" turns off automatically.)

**Comment by ocalhoun6** [comment]
You can set a shortcut key to toggle path issues mode. I have it set to the `-` key, conveniently right next to the number keys that mostly control the more common view transparency/visibility toggles.

------

Or maybe the better approach is that in addition to closing all open windows, the Shift + Backspace shortcut should revert to the default view visibility settings? So that Shift + Backspace always takes you to a 'clean slate' to start the next task.
